sources/api/simplefin.py

```
@@ -0,0 +1,117 @@
+
+        rows = []
+        for account in data.get("accounts", []):
+            for txn in account.get("transactions", []):
+                posted = txn.get("posted", 0)
+                if not posted:
+                    # Pending or unposted transaction; skip — it'll settle and
+                    # appear as posted in a later fetch.
+                    continue
+                rows.append(
+                    {
+                        "date": datetime.datetime.fromtimestamp(posted).date(),
```

Ruff (0.15.17)

[warning] `datetime.datetime.fromtimestamp()` called without a `tz` argument

(DTZ006)